hyperchains.tex

```diff
@@ -1,38 +1,38 @@
 \section{Hyperchains}
 
-The previous approaches had a lot to offer, but considering they cons it is hard
-to scale them in a reasonable way. PoW seems to work well only with big
-computational effort being burned and PoS suffers from huge amount of security
+The previous approaches had a lot to offer, but considering the cons it is hard
+to scale them reasonably. PoW seems to work well only with big
+computational effort being burned and PoS suffers from a huge amount of security
 holes that require very complicated algorithms that usually either don't solve
-the problem at all or just move it further to another layer of abstraction.
+the problem at all or move it further to another layer of abstraction.
 
-Here we present a hybrid strategy which will benefit from stability of PoW
+Here we present a hybrid strategy that will benefit from the stability of PoW
 solutions but will offer the scalability of PoS systems. A Hyperchain is a
 special kind of blockchain that sticks to an already existing chain. They are
-going to be called respectively child and parent chain\cite{hyperchains}.
+going to be called child and parent chain respectively\cite{hyperchains}.
 
 The parent chain can be almost any blockchain in the world. In general, we want
 to use some big existing PoW based chains (at the time of writing, preferably
 Bitcoin or Ethereum, but not limited to) to reuse their burned work to maintain
 the stability of the child chain. We would also like to have
-PoS-like election system to choose the leaders on the hyperchain. In this case
-however we have a very reliable – and mostly important, unpredictable – source
-of randomness – the state of the parent chain. The idea is not very new
+PoS-like election system to choose the leaders on the hyperchain. In this case,
+however, we have a very reliable – and most important, unpredictable – source
+of randomness – the state of the parent chain. The idea is not very new,
 though – there is already some research made in this direction\cite{blockchain_random}.
 
-Having this machinery it seems natural to start a new election each time a
+Having this machinery, it seems natural to start a new election each time a
 (key)block was mined on the parent chain. The next leader shall be chosen
 depending on the hash of that block and selected with chances proportional to
-their stake. The selection algorithm is an abstract over this document – it is
+their stake. The selection algorithm is abstract over this document – it is
 up to the hyperchain to define the details.
 
 $$Insert\ some\ nice\ image\ here$$
 
-We define a group of leadership candidates called "delegates". Each delegate
+We define a group of leadership candidates called "delegates." Each delegate
```

Comment:
```suggestion
We define a group of leadership candidates called "delegates". Each delegate
```

Reply:
Why?

Reply:
I think commas should be inside quotes: https://www.grammarly.com/blog/quotation-marks/

Reply:
Maybe it would be better to use `\emph` instead of quotes?

Reply:
Maybe in this case it will work well, but in the others let's stick to quotes. I was not aware of such convention, looks strange to me. But if it is the right way – we should go for it.